papermill/tests/test_cli.py

```diff
@@ -32,3 +32,16 @@ def test_resolve_type(test_input, expected):
 ])
 def test_is_float(value, expected):
     assert (_is_float(value)) == expected
+
+
+@pytest.mark.parametrize("value,expected", [
+    (13.71, True),
+    ("False", False),
+    ("None", False),
+    (-8.2, True),
+    (10, True),
+    ("hello world", False),
+    ("😍", False),
+])
```

Comment:
Heh, well done.

Reply:
Thanks! Wanted to start with something simple. Will try to add more soon!